packages/babel-plugin-jsx-dom-expressions/README.md

```diff
@@ -1,273 +1,296 @@
+
+### delegateEvents
+
+- Type: `boolean`
+- Default: `true`
+
+Boolean to indicate whether to enable automatic event delegation on camelCase.
+
+### wrapConditionals
+
+- Type: `boolean`
+- Default: `true`
+
+Boolean indicates whether smart conditional detection should be used. This optimizes simple boolean expressions and ternaries in JSX.
+
+### contextToCustomElements
+
+- Type: `boolean`
+- Default: `false`
+
+Boolean indicates whether to set current render context on Custom Elements and slots. Useful for seemless Context API with Web Components.
+
+### builtIns
+
+- Type: `boolean`
+- Default: `string[]`
+
+Array of Component exports from module, that aren't included by default with the library. This plugin will automatically import them if it comes across them in the JSX.
+
+### effectWrapper
+
+- Type: `string`
+- Default: `effect`
+
+This plugin leverages a heuristic for reactive wrapping and lazy evaluation of JSX expressions. This option indicates the reactive wrapper function name (`effect`), defaults to `effect`.
+
+### staticMarker
+
+- Type: `string`
+- Default: `@once`
+
+Comment decorator string indicates the static expression, used to tell the compiler not to wrap them by `effect` function, defaults to `@once`.
+
+### memoWrapper
+
+- Type: `string`
+- Default: `memo`
+
+Memos let you efficiently use a derived value in many reactive computations. This option indicates the memo function name, defaults to `memo`.
+
+### validate
+
+- Type: `boolean`
+- Default: `true`
+
+Checks for properly formed HTML by checking for elements that would not be allowed in certain parent elements. This validation isn't complete but includes places where browser would "correct" it and break the DOM walks.
+
+### omitNestedClosingTags
+
+- Type: `boolean`
+- Default: `false`
+
+Removes unnecessary closing tags from the template output. This may not work in all browser-like environments the same. The solution has been tested again Chrome/Edge/Firefox/Safari.
+
+### omitLastClosingTag
+
+- Type: `boolean`
+- Default: `true`
+
+Removes tags from the template output if they have no closing parents and are the last element. This may not work in all browser-like environments the same. The solution has been tested again Chrome/Edge/Firefox/Safari.
+
+### omitQuotes
+
+- Type: `boolean`
+- Default: `true`
+
+Removes quotes for html attributes when possible from the template output. This may not work in all browser-like environments the same. The solution has been tested again Chrome/Edge/Firefox/Safari.
+
+### requireImportSource
+
+- Type: `string | false`
+- Default: `false`
+
+When set to a string value, this option restricts JSX transformation to only files that contain a specific JSX import source pragma comment. The plugin will only transform JSX in files that include a comment with `@jsxImportSource` followed by the specified value. If the comment is missing or specifies a different import source, the transformation is skipped for that file.
```

Comment:
Actually I think we should be checking the exact lib here 

https://github.com/ryansolid/dom-expressions/blob/c9857a7853ccced5551e9b2ad3d7f284b9eb601c/packages/babel-plugin-jsx-dom-expressions/src/shared/preprocess.js#L37

Reply:
Probably discussed somewhere but we also should be skipping files that `jsxImportSource`-s other lib regardless of `requireImportSource` option which is another thing.

Reply:
> Probably discussed somewhere but we also should be skipping files that `jsxImportSource`-s other lib regardless of `requireImportSource` option which is another thing.

I'm not sure I'm familiar enough with the project to respond meaningfully to this. Do you think that falls within the scope of this PR?

Reply:
Yeah, not directly related to this PR, which allows other plugins to process when this plugin skips if `jsxImportSource` mismatches, which requires `requireImportSource` option. It's about when the option is not set, then, the plugin would process files regardless of `jsxImportSource` pragma, which I think it should be skipping instead.

For this PR, current changes should suffice 👍.